app/assets/stylesheets/thredded/components/_topic-header.scss

```diff
@@ -60,3 +60,67 @@
   font-size: $thredded-font-size-small;
   color: $thredded-secondary-text-color;
 }
+
+
+@mixin following {
+  &::after {
+    display: inline-block;
+    font-size: 1em;
+    width: 1.4rem;
+    height: 1.4rem;
+    position: absolute;
+    top: 0;
+    right: -1.6rem;
+    background-image: image-url("thredded/follow.svg");
```

Comment:
Yes, that's exactly it. You can define an svg inline (you can use the `inline_svg` helper for this, https://github.com/jamesmartin/inline_svg), make it invisible with `display: none`, and `<use>` instances of it in the page with whatever CSS styles needed.